profile/README.md

```diff
@@ -0,0 +1,12 @@
+Valkey is an open source, in memory datastore released under the [BSD-3 Clause License](https://github.com/valkey-io/valkey?tab=BSD-3-Clause-1-ov-file#readme). It is a continuation of the work that was being done on Redis 7.2.4. 
+
+## Community 🎉
+
+- 🎙️ Chat with us on [Discord](https://discord.gg/zbcPa5umUB)
```

Comment:
I'm wondering if we should point to the slack or discord. Thoughts @madolson ?

Reply:
Why not both? I didn't see reference to slack anywhere so I just linked what was available. Do you expect one or the other to go away?

Reply:
Added the slack link in.